CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to Open Source Studio at ITP
+
+## Who can contribute?
+**Everyone** is welcome to engage in the open source project that is the organisation you are currently viewing. We encourage discussion and contribution from outside the student body of the class. 
+
+The only limitation as an outside collaborator is the actual class meeting space, a seminar-style meeting in person. This is a private space for the enrolled students to discuss, question, and collaborate on assignments around the practice and philosophy of open source.
+
+## How to contribute
+Please consider our [code of conduct](https://github.com/Open-Source-Studio-at-ITP/Playground/blob/source/CODE_OF_CONDUCT.md) in all your interactions with our project. 
+
+### Substantive changes
+In general, before making a pull request that alters the *substance* of a document (e.g., changing the terms of the licence or adding rules to a code of conduct), discuss the change first by opening an issue, or commenting on an open issue. Possible exceptions include if the change was discussed in class, or if you are Dan.
```

Comment:
I think this should apply to me as well! Or given that someone else could be the teacher or maintainer of this organization in the future, this language could be more generic,.

Reply:
I think `issue templates` would be involved in this section? Seeing that the topic is scheduled to [week 5](https://github.com/Open-Source-Studio-at-ITP/Syllabus#week-5-documentation-and-accesseibility-oct-2), this might be done later in the future I guess.

Reply:
Yes, we should create issue templates! I think this can be addressed by opening a separate issue to track. I'll do that now.